src/pages/ReviewPage.tsx

```diff
@@ -222,7 +218,8 @@ class ReviewPage extends React.Component<RouteComponentProps, State> {
     this.updateContestChoicesScrollStates()
     window.addEventListener('resize', this.updateContestChoicesScrollStates)
     const targetElement = location.hash && document.querySelector(location.hash)
-    if (targetElement) {
+    /* istanbul ignore next: Tested by Cypress */
```

Comment:
not required for this merge, but I would love if we still unit-tested with 100% test coverage, and kept cypress for end-to-end testing.

Reply:
Checking if something is visible in the browser window is a visual feature. Because **jsdom** is headless, this can not be tested with a unit-test.

Reply:
yeah totally get that you can't test the functionality without cypress, but I like the idea of unit tests at least exercising every path and at least not crashing. Again, not blocking.

Reply:
Cool. Yeah, jsdom — and thus jest too — were crashing when could not run `targetElement.scrollIntoView()`.